presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -487,6 +487,26 @@ public void createPartitionedTableAs()
         }
     }
 
+    @Test
+    public void testShowProperties()
+    {
+        String tableName = "test_show_properties";
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE " + tableName +
+                " WITH (" +
+                "format = 'orc', " +
+                "partitioned_by = ARRAY[ 'SHIP_PRIORITY', 'ORDER_STATUS' ]," +
+                "orc_bloom_filter_columns = ARRAY[ 'SHIP_PRIORITY', 'ORDER_STATUS' ]," +
+                "orc_bloom_filter_fpp = 0.5" +
+                ") " +
+                "AS " +
+                "SELECT orderkey AS order_key, shippriority AS ship_priority, orderstatus AS order_status " +
+                "FROM tpch.tiny.orders";
+
+        assertUpdate(createTable, "SELECT count(*) from orders");
+        assertQuery("select \"orc.bloom.filter.columns\", \"orc.bloom.filter.fpp\" from \"" + tableName + "$properties\"", "select 'ship_priority,order_status', 0.5");
```

Comment:
Can you make this `SELECT *` so we verify all of the columns? Also, uppercase SQL keywords like `SELECT` and `FROM` in the test queries.